record-and-playback/deploy.sh

```diff
@@ -95,10 +97,10 @@ if [ ! -d "$REC_STATUS_SANITY_DIR" ]; then
   sudo mkdir -p $REC_STATUS_SANITY_DIR
 fi
 
-sudo mv /usr/local/bigbluebutton/core/scripts/*.nginx /etc/bigbluebutton/nginx/
-sudo service nginx reload
+#sudo mv /usr/local/bigbluebutton/core/scripts/*.nginx /etc/bigbluebutton/nginx/
```

Comment:
I can pretty much hear @kepstin say "Don't commit commented out lines"

Reply:
All done, I've just made a commit removing these comments, and the one above.